app/kumactl/cmd/install/testdata/install-metrics.defaults.golden.yaml

```diff
@@ -4847,6 +4849,8 @@ spec:
       labels:
         component: "node-exporter"
         app: prometheus
+      annotations:
+        kuma.io/sidecar-injection: "disabled" # disabled for now, injecting DP crashes K8S cluster
```

Comment:
Hmm, is this specific to Kind or Minikube has this problem too? Is this something we want to take a second look at?

Reply:
It crashed both Minikube and Kind. Maybe it's a matter of DeamonSet

Reply:
Please file an issue with the information you have.